src/Stores/task.ts

```diff
@@ -0,0 +1,61 @@
+import { ref } from "vue";
+import { defineStore } from "pinia";
+import type { Task } from "@/type/home";
+
+export const useTaskStore = defineStore("task", () => {
+  const taskToEdit = ref<Task | null>(null);
+
+  const updateTask = async (id: string, params: { name: string; estimatedTime: string }) => {
+    try {
+      const response = await fetch(`http://localhost:3000/tasks/${id}`, {
+        method: "PATCH",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          name: params.name,
+          estimatedTime: params.estimatedTime,
+        }),
+      });
+
+      if (!response.ok) {
+        console.error(`Failed with status: ${response.status}`);
+        return false;
+      }
+
+      const data = await response.json();
+      console.log("Updated task:", data);
+
+      return true;
```

Comment:
why we need to return true?

Reply:
If the function is triggered, it always returns true. It's an incorrect method. Please go through the functions documentation. Please understand what a function is.